deployment/ccip/changeset/deployer_group.go

```diff
@@ -278,8 +278,8 @@ func (d *DeployerGroup) enactDeployer() (deployment.ChangesetOutput, error) {
 				if err != nil {
 					return deployment.ChangesetOutput{}, fmt.Errorf("failed to send transaction: %w", err)
 				}
-
-				_, err = d.e.Chains[selector].Confirm(tx)
+				// TODO how to pass abi here to decode error reason
```

Comment:
Since we have the address book, maybe we could leverage the `To` address of the transaction to know what is the contract were calling and then use that information to load the proper ABI. It's a bit of roundabout way but could probably work